xlators/cluster/afr/src/afr-common.c

```diff
@@ -5321,9 +5361,9 @@ afr_lk(call_frame_t *frame, xlator_t *this, fd_t *fd, int32_t cmd,
         }
     }
 
-    STACK_WIND_COOKIE(frame, afr_lk_cbk, (void *)(long)0, priv->children[i],
-                      priv->children[i]->fops->lk, fd, cmd, flock,
-                      local->xdata_req);
+    local->child_index = i;
```

Comment:
`i` doesn't seem to be used anywhere else. Wouldn't be cleaner to just use `0` instead of the variable ?

Reply:
I will correct it.
Thank you!